axon.go

```diff
@@ -164,6 +166,90 @@ func (a *Axon) Run() {
 	}
 }
 
+func (a *Axon) Verify(schemas, includeTables, excludeTables []string) error {
+
+	if a.Logger == nil {
+		a.Logger = logrus.New()
+		a.Logger.SetFormatter(&logrus.JSONFormatter{})
+	}
+
+	sourceDBConn, err := pgx.Connect(pgx.ConnConfig{
+		Host:     a.Config.SourceDBHost,
+		Port:     uint16(a.Config.SourceDBPort),
+		User:     a.Config.SourceDBUser,
+		Password: a.Config.SourceDBPass,
+		Database: a.Config.SourceDBName,
+	})
+	if err != nil {
+		return fmt.Errorf("unable to connect to source database: %w", err)
+	}
+	err = db.PrepareForDataIntegrityChecks(sourceDBConn)
```

Comment:
Nit: I think it would be easier to read if the is another line separating func calls. However this is not necessary for the err conditionals. It prevents code sneaking in between. Some linters will complain about this.

Reply:
Done, agree, definitely reads better! 😁